docs/tutorials/cli-tool.md

```diff
@@ -46,17 +46,17 @@ You may need to prefix these commands (`sudo npm link`) to run them.
 
 ## Launching the CLI tool
 
-1. Navigate to the `/core` folder: `cd core`
+1. Navigate to the `packages/core` folder: `cd packages/core`
```

Comment:
Can all of the steps below be done from root?

Reply:
ye, I think they should be able to be run from root. I've refactored it to reflect this. take a look & LMK.